Update Endpoints Connectors Easyvista Service Manager

ezv-compteintegration opened the pull request:
Add new Endpoints Connectors Easyvista Service Manager

ezv-compteintegration committed:
Add files via upload

Amjed-Ayoub commented:
@ezv-compteintegration thank you so much for open sourcing your connector. May I ask you please to unzip the files in this PR. please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors).

ezv-compteintegration commented:
> @ezv-compteintegration thank you so much for open sourcing your connector. May I ask you please to unzip the files in this PR. please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors).

@Amjed-Ayoub Hello can you tell how to UNZIP the ZIP file in this PR. Because i can't upload JSON file in comment section. In your process your ask a ZIP file this is why i gave you a ZIP file n the PR

Amjed-Ayoub commented:
> > @ezv-compteintegration thank you so much for open sourcing your connector. May I ask you please to unzip the files in this PR. please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors).
> 
> @Amjed-Ayoub Hello can you tell how to UNZIP the ZIP file in this PR. Because i can't upload JSON file in comment section. In your process your ask a ZIP file this is why i gave you a ZIP file n the PR

@ezv-compteintegration please follow the instructions in our [readme file](https://github.com/microsoft/PowerPlatformConnectors#certified-connectors) in the [Contributing section](https://github.com/microsoft/PowerPlatformConnectors#contributing).  please [create your own fork](https://github.com/microsoft/PowerPlatformConnectors#creating-a-fork) from this repo.

Then using the cmd/terminal you can install git and add your files or you can use GitHub for Desktop. 

Please let me know if you have any questions.

ezv-compteintegration committed:
add file from ZIP unzip

add file from ZIP unzip

ezv-compteintegration commented:
@Amjed-Ayoub I add the 3 files with Github Desktop

ezv-compteintegration commented:
@Amjed-Ayoub 
@microsoft-github-policy-service  agree company="Easyvista"

ezv-compteintegration commented:
@microsoft-github-policy-service agree company="Easyvista"

Amjed-Ayoub commented:
@ezv-compteintegration you are adding the files at the main root. please create a folder with your connector name in the "certified-connectors" subfolder :-) and add your files there. 
In this PR you are making changes to our main readme file. Please revert back the changes.

ezv-compteintegration committed:
Revert "add file from ZIP unzip"

This reverts commit 47a338acaeb48843c9d31fa0ebab740ff9f0df07.

ezv-compteintegration committed:
Add file in the Sub Folder

Add file in the Sub Folder named "Easyvista_Service._Manager"

ezv-compteintegration commented:
@Amjed-Ayoub I revert back the changes and I add file in the Sub Folder named "Easyvista_Service._Manager"

Amjed-Ayoub commented on certified-connectors/Easyvista_Service._Manager/Readme.md:
```diff
@@ -0,0 +1,277 @@
+Connector Easyvista Service Manager
```
@ezv-compteintegration Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)for the readme file. You can see the raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

Amjed-Ayoub replied:
@ezv-compteintegration Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)for the readme file. You can see the raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

Amjed-Ayoub commented on certified-connectors/Easyvista_Service._Manager/apiProperties.json:
```diff
@@ -0,0 +1,54 @@
+        "title": "Dynamic Host URL"
+      }
+    ]
+  }
```
@ezv-compteintegration Please add the "publisher" and "stackowner" properties.
```
"publisher": "Name of company or organization.",
"stackOwner": "Name of the owner of the service or product."
```
Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).

Amjed-Ayoub commented on certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,13162 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "EasyVista Service Manager Modification Décembre 2022",
```
@ezv-compteintegration I would highly recommend a better description for the connector in English as this will show in the connector [documentation](https://learn.microsoft.com/en-us/connectors/easyvista/).

Amjed-Ayoub reviewed (changes requested):
@ezv-compteintegration the swagger file has the below errors. May I ask you please to resolve them all. 
```
##[error]The 'description' value must be written in English. ValuePath: /info/description
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1departments~1{department_id}/put/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1departments~1{department_id}/put/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1departments~1{department_id}/put/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1departments~1{department_id}/put/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1locations~1{location_id}/put/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1locations~1{location_id}/put/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1locations~1{location_id}/put/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1locations~1{location_id}/put/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1asset-characteristics/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1asset-characteristics/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1asset-characteristics~1{document_id}/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1asset-characteristics~1{document_id}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1asset-characteristics~1{document_id}/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1asset-characteristics~1{document_id}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/put/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/put/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/put/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/put/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/put/parameters/2
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/put/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/post/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/post/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/post/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/post/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/post/parameters/2
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1assets~1{asset_id}~1characteristics~1{characteristic_id}/post/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~150006~1assets~1{ci_id}/put/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~150006~1assets~1{ci_id}/put/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~150006~1configuration-items~1{ci_id}/put/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~150006~1configuration-items~1{ci_id}/put/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~150006~1configuration-items~1{ci_id}~1item-links~1impacting/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~150006~1configuration-items~1{ci_id}~1item-links~1impacting/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~150006~1configuration-items~1{ci_id}~1item-links~1impacted/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~150006~1configuration-items~1{ci_id}~1item-links~1impacted/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1status/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1status/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problem-links/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problem-links/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1requests~1{rfc_number}~1problems/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1requests~1{rfc_number}~1problems/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1requests~1{rfc_number}~1problems/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1requests~1{rfc_number}~1problems/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems~1{rfc_number}~1requests/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems~1{rfc_number}~1requests/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems~1{rfc_number}~1requests/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems~1{rfc_number}~1requests/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1documents~1{document_id}/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1documents~1{document_id}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1documents~1{document_id}/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1documents~1{document_id}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions~1{question_id}/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions~1{question_id}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions~1{question_id}/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions~1{question_id}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1~1questions-result~1{request_id}/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1~1questions-result~1{request_id}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1~1questions-result~1{request_id}/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1~1questions-result~1{request_id}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/get/parameters/2
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/get/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/put/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/put/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/post/parameters/2
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questions-result~1{request_id}~1{question_id}/post/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questionnaires/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questionnaires/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questionnaires~1{questionnaire_id}/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questionnaires~1{questionnaire_id}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questionnaires~1{questionnaire_id}/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1questionnaires~1{questionnaire_id}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems~1{rfc_number}/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems~1{rfc_number}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems~1{rfc_number}/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1problems~1{rfc_number}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news/post/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news/post/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news~1{document_id}/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news~1{document_id}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news~1{document_id}/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news~1{document_id}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news~1{document_id}/put/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news~1{document_id}/put/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news~1{document_id}/put/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1news~1{document_id}/put/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1actions/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1actions/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1requests~1{rfc_number}~1actions/post/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1requests~1{rfc_number}~1actions/post/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1requests~1{rfc_number}~1actions/post/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1requests~1{rfc_number}~1actions/post/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE/post/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE/post/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE~1{e_table_id}/put/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE~1{e_table_id}/put/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE~1{e_table_id}/put/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE~1{e_table_id}/put/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE~1{e_table_id}/delete/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE~1{e_table_id}/delete/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE~1{e_table_id}/delete/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1~1{account}~1E_TABLE~1{e_table_id}/delete/parameters/1
##[error]The 'contact' property is required. ValuePath: /info
##[error]The 'x-ms-connector-metadata' property is required.

```

Amjed-Ayoub commented:
@ezv-compteintegration Please remove the zip folder from this PR. Thank you so much.

ezv-compteintegration committed:
Delete Git_shared_easyvista.20service.20manager.5fc68cafe785a1947-801708a00708f80c.zip

ezv-compteintegration committed:
New files

New files

ezv-compteintegration commented:
@Amjed-Ayoub 

I remove the zip folder
i modified apiProperties.json
i modified readme.md

But i don't understand the errors in apiDefinition.swagger.json on:
"description" and "x-ms-summary" because all my endpoints already have these elements filled
in the swagger file you can watch that i have aleready these informations

And i don't know what is :
##[error]The 'contact' property is required. ValuePath: /info
##[error]The 'x-ms-connector-metadata' property is required.

Can you help me ?

Amjed-Ayoub commented on certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,13162 @@
+                },
+                "COST_PER_HOUR": {
+                  "description": "COST_PER_HOUR",
+                  "title": "COST_PER_HOUR
```
@ezv-compteintegration it looks like you have mistakenly added the operation "summary", "description", "operationId", and "x-ms-visibility" inside of the parameters. please move it to the post operation scope.

Amjed-Ayoub commented on certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,13162 @@
+  "info": {
+    "description": "Different endpoints for creating/updating/deleting in the Easyvista Service Manager database / EasyVista Service Manager Modification May 2023",
+    "title": "EasyVista Service Manager",
+    "version": "1.0.0"
```
@ezv-compteintegration in the "info" section please add the "contact" property. Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json). 
```
    "contact": {
      "name": "Full name",
      "url": "Contact URL",
      "email": "Contact email address"
    }
```

Amjed-Ayoub commented on certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,13162 @@
+                },
+                "COST_PER_HOUR": {
+                  "description": "COST_PER_HOUR",
+                  "title": "COST_PER_HOUR
```
@ezv-compteintegration at the end of the file, please add the "x-ms-connector-metadata". Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json).
```
  "x-ms-connector-metadata": [
    {
      "propertyName": "Website",
      "propertyValue": ""
    },
    {
      "propertyName": "Privacy policy",
      "propertyValue": ""
    },
    {
      "propertyName": "Categories",
      "propertyValue": ""
    }
  ]
```

ezv-compteintegration replied:
@Amjed-Ayoub 

I modified the whole apiDefinition.swagger.json file

Amjed-Ayoub replied:
Hello @ezv-compteintegration,

Thank you so much for getting back to me. Unfortunately the PR is still having a lot of errors.
![image](https://github.com/microsoft/PowerPlatformConnectors/assets/97551789/3f0763e5-3fb5-4c98-9cb1-cc7007484bce)

I would be happy to schedule a quick call with you to discuss this in detail. May I ask you please to send us an email to our Connector Partner Mgmt Team <ConnectorPartnerMgmtTeam@service.microsoft.com>, and I will make sure to respond quickly. 

Thank you so much again.

ezv-compteintegration replied:
Hello,
yes we can. In fact, it's my colleague who make all the changes but now he is not here.
Thanks, my english is not very well but we can try

ezv-compteintegration replied:
@Amjed-Ayoub file modified, now no errors ;)

ezv-compteintegration committed:
Update apiDefinition.swagger.json

ezv-compteintegration committed:
Update apiDefinition.swagger.json

ezv-compteintegration committed:
Update apiDefinition.swagger.json

ezv-compteintegration committed:
Update apiDefinition.swagger.json

Amjed-Ayoub commented on certified-connectors/Easyvista_Service._Manager/apiProperties.json:
```diff
@@ -0,0 +1,56 @@
+  "properties": {
+    "capabilities": [],
+    "publisher": "EasyVista",
+    "stackOwner": "Cédric MARTIN / Product Team",
```
@ezv-compteintegration for all our verified partners we ask to match the "publisher" and "stackOwner".
Please change the stackOwner to "EasyVista"

ezv-compteintegration committed:
Update apiDefinition.swagger.json

ezv-compteintegration committed:
Update apiDefinition.swagger.json

ezv-compteintegration committed:
Modifications

ezv-compteintegration commented:
@Amjed-Ayoub  I made the changes in [apiProperties.json] & [Readme.md]
I also made a lot of change in [apiDefinition.swagger.json] for your excel file (localized) but i don't know how find :

Line 475 to 662 of your excel files :

0; "(...)   ies.HREF.description" | $.records.[*].CATALOG_ASSET.HREF
0; "(...)   .DEPARTMENT_CODE.description" | DEPARTMENT_CODE
0; "(...)   .DEPARTMENT_LABEL.description" | DEPARTMENT_LABEL
0; "(...)   .DEPARTMENT_PATH.description" | $.records.[*].DEPARTMENT.DEPARTMENT_PATH
0; "(...)   EPARTMENT_PATH.description" | $.records.[*].EMPLOYEE.DEPARTMENT_PATH
0; "(...) REF.description" | $.records.[*].EMPLOYEE.HREF
0; "(...) REF.description" | $.records.[*].LOCATION.HREF
0; "(...) CT.description" | EMPLOYEE_BEGIN_OF_CONTRACT
0; "(...) .description" | EMPLOYEE_CELLULAR_NUMBER
0; "(...) .description" | EMPLOYEE_DEPARTMENT_PATH
0; "(...) escription" | EMPLOYEE_LOCATION_PATH
0; "(...) scription" | EMPLOYEE_PHONE_NUMBER
0; "(...)   s.CONTRACT_ROW.description" | CONTRACT_ROW
0; "(...)   s.MONTHLY_PAYMENT.description" | MONTHLY_PAYMENT
0; "(...)   s.PARENT_ASSET_ID.description" | PARENT_ASSET_ID
0; "(...)   properties.HREF.description" | $.records.[*].MANUFACTURER.HREF
0; "(...) escription" | CURRENT_LICENSE_VERSION
0; "(...) scription" | DEFAULT_BUY_BACK_VALUE
0; "(...) R_ID.description" | DEFAULT_BUY_BACK_VALUE_CUR_ID
0; "(...) D.description" | DEFAULT_CHARGE_BACK_CUR_ID
0; "(...) scription" | DEFAULT_DISPOSAL_VALUE
0; "(...) R_ID.description" | DEFAULT_DISPOSAL_VALUE_CUR_ID
0; "(...) cription" | DEFAULT_RENEWAL_VALUE
0; "(...) _ID.description" | DEFAULT_RENEWAL_VALUE_CUR_ID
0; "(...) .HREF.description" | DESCRIPTION_EN_HREF
0; "(...) .HREF.description" | DESCRIPTION_FR_HREF
0; "(...) .HREF.description" | DESCRIPTION_GE_HREF
0; "(...) .HREF.description" | DESCRIPTION_IT_HREF
0; "(...) .HREF.description" | DESCRIPTION_L1_HREF
0; "(...) .HREF.description" | DESCRIPTION_L2_HREF
0; "(...) .HREF.description" | DESCRIPTION_L3_HREF
0; "(...) .HREF.description" | DESCRIPTION_L4_HREF
0; "(...) .HREF.description" | DESCRIPTION_L5_HREF
0; "(...) .HREF.description" | DESCRIPTION_L6_HREF
0; "(...) .HREF.description" | DESCRIPTION_PO_HREF
0; "(...) .HREF.description" | DESCRIPTION_SP_HREF
0; "(...) ON_WH.description" | ESTIMATED_POWER_CONSUMPTION_WH
0; "(...) scription" | LICENSE_PRICE_PER_SEAT
0; "(...) R_ID.description" | LICENSE_PRICE_PER_SEAT_CUR_ID
0; "(...)   ISCOVERY_NAME.description" | MANUFACTURER_DISCOVERY_NAME
0; "(...) REF.description" | MANUFACTURER_HREF
0; "(...) ATION.description" | MANUFACTURER_WARRANTY_DURATION
0; "(...) description" | PERCENT_MAINTENANCE_COST
0; "(...) IRED.description" | TECHNICAL_VALIDATION_REQUIRED
0; "(...)   TALOG_PATH_FR.description" | SD_CATALOG_PATH_FR
0; "(...) N.description" | SD_CATALOG_PATH_EN
0; "(...) E.description" | SD_CATALOG_PATH_GE
0; "(...) T.description" | SD_CATALOG_PATH_IT
0; "(...) 1.description" | SD_CATALOG_PATH_L1
0; "(...) 2.description" | SD_CATALOG_PATH_L2
0; "(...) 3.description" | SD_CATALOG_PATH_L3
0; "(...) 4.description" | SD_CATALOG_PATH_L4
0; "(...) 5.description" | SD_CATALOG_PATH_L5
0; "(...) 6.description" | SD_CATALOG_PATH_L6
0; "(...) O.description" | SD_CATALOG_PATH_PO
0; "(...) P.description" | SD_CATALOG_PATH_SP
0; "(...)   UESTS_PATH.description" | CATALOG_REQUESTS_PATH
0; "(...)   UESTS_PATH.properties.HREF.description" | $.records.[*].CATALOG_REQUESTS_PATH.HREF
0; "(...)   UESTS_PATH.properties.SD_CATALOG_ID.description" | $.records.[*].CATALOG_REQUESTS_PATH.SD_CATALOG_ID
0; "(...)   UEST_PATH.description" | CATALOG_REQUEST_PATH
0; "(...) ID.description" | $.records.[*].SD_CATALOG_ID
0; "(...)   roperties.HREF.description" | CATALOG_REQUEST_PATH_HREF
0; "(...)   roperties.SD_CATALOG_ID.description" | CATALOG_REQUEST_PATH_SD_CATALOG_ID
0; "(...) tion" | COMMENT_CATALOG
0; "(...) ies.HREF.description" | COMMENT_CATALOG_HREF
0; "(...)   N_OF_CONTRACT.description" | MANAGER_BEGIN_OF_CONTRACT
0; "(...)   ULAR_NUMBER.description" | MANAGER_CELLULAR_NUMBER
0; "(...)   RTMENT_PATH.description" | MANAGER_DEPARTMENT_PATH
0; "(...) OYEE_ID.description" | MANAGER_EMPLOYEE_ID
0; "(...) IL.description" | MANAGER_E_MAIL
0; "(...) _NAME.description" | MANAGER_LAST_NAME
0; "(...)   TION_PATH.description" | MANAGER_LOCATION_PATH
0; "(...) E_NUMBER.description" | MANAGER_PHONE_NUMBER
0; "(...) description" | SLA_NAME_FR
0; "(...)   properties.BLOCKING.description" | BLOCKING
0; "(...)   properties.CHILD_CI_ID.description" | CHILD_CI_ID
0; "(...)   properties.PARENT_CI_ID.description" | PARENT_CI_ID
0; "(...)   properties.RELATION_TYPE.description" | RELATION_TYPE
0; "(...)   properties.RELATION_TYPE.properties.REFERENCE_FR.description" | REFERENCE_FR
0; "(...)   properties.RELATION_TYPE.properties.REFERENCE_ID.description" | REFERENCE_ID
0; "(...)   properties.RELATION_TYPE_ID.description" | RELATION_TYPE_ID
0; "(...) ption" | COMMENT_DEPARTMENT
0; "(...) ription" | PARENT_DEPARTMENT_ID
0; "(...) scription" | PARENT_DEPARTMENT_PATH
0; "(...) .HREF.description" | MANAGER_HREF
0; "(...) REF.description" | COMMENT_EMPLOYEE_HREF
0; "(...) REF.description" | $.COMMENT_LOCATION.HREF
0; "(...)   perties.HREF.description" | $.records.[*].CATALOG_REQUEST.HREF
0; "(...) HREF.description" | $.records.[*].COMMENT.HREF
0; "(...) es.HREF.description" | $.records.[*].DEPARTMENT.HREF
0; "(...)   rties.HREF.description" | $.records.[*].KNOWN_PROBLEM.HREF
0; "(...)   .LOCATION_PATH.description" | $.records.[*].LOCATION.LOCATION_PATH
0; "(...)   s.BEGIN_OF_CONTRACT.description" | $.records.[*].RECIPIENT.BEGIN_OF_CONTRACT
0; "(...)   s.CELLULAR_NUMBER.description" | $.records.[*].RECIPIENT.CELLULAR_NUMBER
0; "(...)   s.DEPARTMENT_PATH.description" | $.records.[*].RECIPIENT.DEPARTMENT_PATH
0; "(...)   s.EMPLOYEE_ID.description" | $.records.[*].RECIPIENT.EMPLOYEE_ID
0; "(...) s.E_MAIL.description" | $.records.[*].RECIPIENT.E_MAIL
0; "(...) s.HREF.description" | $.records.[*].RECIPIENT.HREF
0; "(...)   s.LAST_NAME.description" | $.records.[*].RECIPIENT.LAST_NAME
0; "(...)   s.LOCATION_PATH.description" | $.records.[*].RECIPIENT.LOCATION_PATH
0; "(...)   s.PHONE_NUMBER.description" | $.records.[*].RECIPIENT.PHONE_NUMBER
0; "(...)   s.BEGIN_OF_CONTRACT.description" | $.records.[*].REQUESTOR.BEGIN_OF_CONTRACT
0; "(...)   s.CELLULAR_NUMBER.description" | $.records.[*].REQUESTOR.CELLULAR_NUMBER
0; "(...)   s.DEPARTMENT_PATH.description" | $.records.[*].REQUESTOR.DEPARTMENT_PATH
0; "(...)   s.EMPLOYEE_ID.description" | $.records.[*].REQUESTOR.EMPLOYEE_ID
0; "(...) s.E_MAIL.description" | $.records.[*].REQUESTOR.E_MAIL
0; "(...) s.HREF.description" | $.records.[*].REQUESTOR.HREF
0; "(...)   s.LAST_NAME.description" | $.records.[*].REQUESTOR.LAST_NAME
0; "(...)   s.LOCATION_PATH.description" | $.records.[*].REQUESTOR.LOCATION_PATH
0; "(...)   s.PHONE_NUMBER.description" | $.records.[*].REQUESTOR.PHONE_NUMBER
0; "(...) REF.description" | $.records.[*].STATUS.HREF
0; "(...) TATUS_FR.description" | STATUS_FR
0; "(...)   TATUS_GUID.description" | STATUS_GUID
0; "(...) ription" | CI_Asset_Tag
0; "(...) on" | CI_Name
0; "(...) ription" | Catalog_Code
0; "(...) cription" | ParentRequest
0; "(...) ication.description" | Recipient_Identification
0; "(...) scription" | Recipient_Mail
0; "(...) scription" | Recipient_Name
0; "(...) ication.description" | Requestor_Identification
0; "(...) scription" | Requestor_Mail
0; "(...) scription" | Requestor_Name
0; "(...) .title" | CATALOG_REQUEST_HREF
0; "(...) cription" | DESCRIPTION_HREF
0; "(...) .description" | DYNAMIC_DETAILS_HREF
0; "(...) cription" | EFFECTIVE_CHANGE_DATE_START
0; "(...) escription" | KNOWN_PROBLEM_HREF
0; "(...) ITING.description" | MS_PROJECT_IMPORT_VALIDATION_WAITING
0; "(...) _PATH.description" | RECIPIENT_DEPARTMENT_PATH
0; "(...) D.description" | RECIPIENT_EMPLOYEE_ID
0; "(...) cription" | RECIPIENT_E_MAIL
0; "(...) description" | RECIPIENT_LAST_NAME
0; "(...) ATH.description" | RECIPIENT_LOCATION_PATH
0; "(...) ER.description" | RECIPIENT_PHONE_NUMBER
0; "(...) cription" | REQUESTED_CHANGE_DATE_START
0; "(...) ONTRACT.description" | REQUESTOR_BEGIN_OF_CONTRACT
0; "(...) UMBER.description" | REQUESTOR_CELLULAR_NUMBER
0; "(...) _PATH.description" | REQUESTOR_DEPARTMENT_PATH
0; "(...) D.description" | REQUESTOR_EMPLOYEE_ID
0; "(...) cription" | REQUESTOR_E_MAIL
0; "(...) description" | REQUESTOR_LAST_NAME
0; "(...) ATH.description" | REQUESTOR_LOCATION_PATH
0; "(...) ER.description" | REQUESTOR_PHONE_NUMBER
0; "(...) F.description" | RISK_DESCRIPTION_HREF
0; "(...) escription" | TIME_USED_TO_DELIVER_FEEDBACK
0; "(...)   e_actions.description" | delete_actions
0; "(...)   erties.filedata.description" | filedata
0; "(...)   erties.filename.description" | filename
0; "(...) on" | action_type_id
0; "(...) tion" | contractual_cost
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.description" | folders
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.hidden.title" | folders hidden
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.iconId.title" | folders iconId
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.id.description" | id
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.id.title" | folders id
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.index.title" | folders index
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.name.title" | folders name
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.hidden.description" | hidden
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.iconId.description" | iconId
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.index.description" | index
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.name.description" | name
0;   "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.scenarios.description" | scenarios
0;   "paths./AtanorPortalAPI/atanor/projects/.get.responses.200.schema.properties.domainList.description" | domainList
0;   "paths./AtanorPortalAPI/atanor/projects/.get.responses.200.schema.properties.domainList.items.properties.id.title" | domain id
0;   "paths./AtanorPortalAPI/atanor/projects/.get.responses.200.schema.properties.domainList.items.properties.name.title" | domain name
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.availableScenarios.description" | availableScenarios
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.bestRank.description" | bestRank
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.domainId.description" | domainId
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.locale.description" | locale
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.pattern.description" | pattern
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.scenarios.items.properties.iconId.title" | scenarios iconId
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.scenarios.items.properties.id.title" | scenarios id
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.scenarios.items.properties.name.title" | scenarios name
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.scenarios.items.properties.rank.title" | scenarios rank
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.searchTimeMs.description" | searchTimeMs
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.searchUid.description" | searchUid
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.userId.description" | userId
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.userLogin.description" | userLogin
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.versionId.description" | versionId
0;   "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.worstRank.description" | worstRank
0;   "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.email.description" | email
0;   "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.expert.description" | expert
0;   "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.firstName.description" | firstName
0;   "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.grps.description" | grps
0;   "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.lastName.description" | lastName
0;   "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.properties.description" | properties
0;   "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.sessionLess.description" | sessionLess

Amjed-Ayoub commented:
> @Amjed-Ayoub I made the changes in [apiProperties.json] & [Readme.md] I also made a lot of change in [apiDefinition.swagger.json] for your excel file (localized) but i don't know how find :
> 
> Line 475 to 662 of your excel files :
> 
> 0; "(...) ies.HREF.description" | $.records.[_].CATALOG_ASSET.HREF 0; "(...) .DEPARTMENT_CODE.description" | DEPARTMENT_CODE 0; "(...) .DEPARTMENT_LABEL.description" | DEPARTMENT_LABEL 0; "(...) .DEPARTMENT_PATH.description" | $.records.[_].DEPARTMENT.DEPARTMENT_PATH 0; "(...) EPARTMENT_PATH.description" | $.records.[_].EMPLOYEE.DEPARTMENT_PATH 0; "(...) REF.description" | $.records.[_].EMPLOYEE.HREF 0; "(...) REF.description" | $.records.[_].LOCATION.HREF 0; "(...) CT.description" | EMPLOYEE_BEGIN_OF_CONTRACT 0; "(...) .description" | EMPLOYEE_CELLULAR_NUMBER 0; "(...) .description" | EMPLOYEE_DEPARTMENT_PATH 0; "(...) escription" | EMPLOYEE_LOCATION_PATH 0; "(...) scription" | EMPLOYEE_PHONE_NUMBER 0; "(...) s.CONTRACT_ROW.description" | CONTRACT_ROW 0; "(...) s.MONTHLY_PAYMENT.description" | MONTHLY_PAYMENT 0; "(...) s.PARENT_ASSET_ID.description" | PARENT_ASSET_ID 0; "(...) properties.HREF.description" | $.records.[_].MANUFACTURER.HREF 0; "(...) escription" | CURRENT_LICENSE_VERSION 0; "(...) scription" | DEFAULT_BUY_BACK_VALUE 0; "(...) R_ID.description" | DEFAULT_BUY_BACK_VALUE_CUR_ID 0; "(...) D.description" | DEFAULT_CHARGE_BACK_CUR_ID 0; "(...) scription" | DEFAULT_DISPOSAL_VALUE 0; "(...) R_ID.description" | DEFAULT_DISPOSAL_VALUE_CUR_ID 0; "(...) cription" | DEFAULT_RENEWAL_VALUE 0; "(...) _ID.description" | DEFAULT_RENEWAL_VALUE_CUR_ID 0; "(...) .HREF.description" | DESCRIPTION_EN_HREF 0; "(...) .HREF.description" | DESCRIPTION_FR_HREF 0; "(...) .HREF.description" | DESCRIPTION_GE_HREF 0; "(...) .HREF.description" | DESCRIPTION_IT_HREF 0; "(...) .HREF.description" | DESCRIPTION_L1_HREF 0; "(...) .HREF.description" | DESCRIPTION_L2_HREF 0; "(...) .HREF.description" | DESCRIPTION_L3_HREF 0; "(...) .HREF.description" | DESCRIPTION_L4_HREF 0; "(...) .HREF.description" | DESCRIPTION_L5_HREF 0; "(...) .HREF.description" | DESCRIPTION_L6_HREF 0; "(...) .HREF.description" | DESCRIPTION_PO_HREF 0; "(...) .HREF.description" | DESCRIPTION_SP_HREF 0; "(...) ON_WH.description" | ESTIMATED_POWER_CONSUMPTION_WH 0; "(...) scription" | LICENSE_PRICE_PER_SEAT 0; "(...) R_ID.description" | LICENSE_PRICE_PER_SEAT_CUR_ID 0; "(...) ISCOVERY_NAME.description" | MANUFACTURER_DISCOVERY_NAME 0; "(...) REF.description" | MANUFACTURER_HREF 0; "(...) ATION.description" | MANUFACTURER_WARRANTY_DURATION 0; "(...) description" | PERCENT_MAINTENANCE_COST 0; "(...) IRED.description" | TECHNICAL_VALIDATION_REQUIRED 0; "(...) TALOG_PATH_FR.description" | SD_CATALOG_PATH_FR 0; "(...) N.description" | SD_CATALOG_PATH_EN 0; "(...) E.description" | SD_CATALOG_PATH_GE 0; "(...) T.description" | SD_CATALOG_PATH_IT 0; "(...) 1.description" | SD_CATALOG_PATH_L1 0; "(...) 2.description" | SD_CATALOG_PATH_L2 0; "(...) 3.description" | SD_CATALOG_PATH_L3 0; "(...) 4.description" | SD_CATALOG_PATH_L4 0; "(...) 5.description" | SD_CATALOG_PATH_L5 0; "(...) 6.description" | SD_CATALOG_PATH_L6 0; "(...) O.description" | SD_CATALOG_PATH_PO 0; "(...) P.description" | SD_CATALOG_PATH_SP 0; "(...) UESTS_PATH.description" | CATALOG_REQUESTS_PATH 0; "(...) UESTS_PATH.properties.HREF.description" | $.records.[_].CATALOG_REQUESTS_PATH.HREF 0; "(...) UESTS_PATH.properties.SD_CATALOG_ID.description" | $.records.[_].CATALOG_REQUESTS_PATH.SD_CATALOG_ID 0; "(...) UEST_PATH.description" | CATALOG_REQUEST_PATH 0; "(...) ID.description" | $.records.[_].SD_CATALOG_ID 0; "(...) roperties.HREF.description" | CATALOG_REQUEST_PATH_HREF 0; "(...) roperties.SD_CATALOG_ID.description" | CATALOG_REQUEST_PATH_SD_CATALOG_ID 0; "(...) tion" | COMMENT_CATALOG 0; "(...) ies.HREF.description" | COMMENT_CATALOG_HREF 0; "(...) N_OF_CONTRACT.description" | MANAGER_BEGIN_OF_CONTRACT 0; "(...) ULAR_NUMBER.description" | MANAGER_CELLULAR_NUMBER 0; "(...) RTMENT_PATH.description" | MANAGER_DEPARTMENT_PATH 0; "(...) OYEE_ID.description" | MANAGER_EMPLOYEE_ID 0; "(...) IL.description" | MANAGER_E_MAIL 0; "(...) _NAME.description" | MANAGER_LAST_NAME 0; "(...) TION_PATH.description" | MANAGER_LOCATION_PATH 0; "(...) E_NUMBER.description" | MANAGER_PHONE_NUMBER 0; "(...) description" | SLA_NAME_FR 0; "(...) properties.BLOCKING.description" | BLOCKING 0; "(...) properties.CHILD_CI_ID.description" | CHILD_CI_ID 0; "(...) properties.PARENT_CI_ID.description" | PARENT_CI_ID 0; "(...) properties.RELATION_TYPE.description" | RELATION_TYPE 0; "(...) properties.RELATION_TYPE.properties.REFERENCE_FR.description" | REFERENCE_FR 0; "(...) properties.RELATION_TYPE.properties.REFERENCE_ID.description" | REFERENCE_ID 0; "(...) properties.RELATION_TYPE_ID.description" | RELATION_TYPE_ID 0; "(...) ption" | COMMENT_DEPARTMENT 0; "(...) ription" | PARENT_DEPARTMENT_ID 0; "(...) scription" | PARENT_DEPARTMENT_PATH 0; "(...) .HREF.description" | MANAGER_HREF 0; "(...) REF.description" | COMMENT_EMPLOYEE_HREF 0; "(...) REF.description" | $.COMMENT_LOCATION.HREF 0; "(...) perties.HREF.description" | $.records.[_].CATALOG_REQUEST.HREF 0; "(...) HREF.description" | $.records.[_].COMMENT.HREF 0; "(...) es.HREF.description" | $.records.[_].DEPARTMENT.HREF 0; "(...) rties.HREF.description" | $.records.[_].KNOWN_PROBLEM.HREF 0; "(...) .LOCATION_PATH.description" | $.records.[_].LOCATION.LOCATION_PATH 0; "(...) s.BEGIN_OF_CONTRACT.description" | $.records.[_].RECIPIENT.BEGIN_OF_CONTRACT 0; "(...) s.CELLULAR_NUMBER.description" | $.records.[_].RECIPIENT.CELLULAR_NUMBER 0; "(...) s.DEPARTMENT_PATH.description" | $.records.[_].RECIPIENT.DEPARTMENT_PATH 0; "(...) s.EMPLOYEE_ID.description" | $.records.[_].RECIPIENT.EMPLOYEE_ID 0; "(...) s.E_MAIL.description" | $.records.[_].RECIPIENT.E_MAIL 0; "(...) s.HREF.description" | $.records.[_].RECIPIENT.HREF 0; "(...) s.LAST_NAME.description" | $.records.[_].RECIPIENT.LAST_NAME 0; "(...) s.LOCATION_PATH.description" | $.records.[_].RECIPIENT.LOCATION_PATH 0; "(...) s.PHONE_NUMBER.description" | $.records.[_].RECIPIENT.PHONE_NUMBER 0; "(...) s.BEGIN_OF_CONTRACT.description" | $.records.[_].REQUESTOR.BEGIN_OF_CONTRACT 0; "(...) s.CELLULAR_NUMBER.description" | $.records.[_].REQUESTOR.CELLULAR_NUMBER 0; "(...) s.DEPARTMENT_PATH.description" | $.records.[_].REQUESTOR.DEPARTMENT_PATH 0; "(...) s.EMPLOYEE_ID.description" | $.records.[_].REQUESTOR.EMPLOYEE_ID 0; "(...) s.E_MAIL.description" | $.records.[_].REQUESTOR.E_MAIL 0; "(...) s.HREF.description" | $.records.[_].REQUESTOR.HREF 0; "(...) s.LAST_NAME.description" | $.records.[_].REQUESTOR.LAST_NAME 0; "(...) s.LOCATION_PATH.description" | $.records.[_].REQUESTOR.LOCATION_PATH 0; "(...) s.PHONE_NUMBER.description" | $.records.[_].REQUESTOR.PHONE_NUMBER 0; "(...) REF.description" | $.records.[*].STATUS.HREF 0; "(...) TATUS_FR.description" | STATUS_FR 0; "(...) TATUS_GUID.description" | STATUS_GUID 0; "(...) ription" | CI_Asset_Tag 0; "(...) on" | CI_Name 0; "(...) ription" | Catalog_Code 0; "(...) cription" | ParentRequest 0; "(...) ication.description" | Recipient_Identification 0; "(...) scription" | Recipient_Mail 0; "(...) scription" | Recipient_Name 0; "(...) ication.description" | Requestor_Identification 0; "(...) scription" | Requestor_Mail 0; "(...) scription" | Requestor_Name 0; "(...) .title" | CATALOG_REQUEST_HREF 0; "(...) cription" | DESCRIPTION_HREF 0; "(...) .description" | DYNAMIC_DETAILS_HREF 0; "(...) cription" | EFFECTIVE_CHANGE_DATE_START 0; "(...) escription" | KNOWN_PROBLEM_HREF 0; "(...) ITING.description" | MS_PROJECT_IMPORT_VALIDATION_WAITING 0; "(...) _PATH.description" | RECIPIENT_DEPARTMENT_PATH 0; "(...) D.description" | RECIPIENT_EMPLOYEE_ID 0; "(...) cription" | RECIPIENT_E_MAIL 0; "(...) description" | RECIPIENT_LAST_NAME 0; "(...) ATH.description" | RECIPIENT_LOCATION_PATH 0; "(...) ER.description" | RECIPIENT_PHONE_NUMBER 0; "(...) cription" | REQUESTED_CHANGE_DATE_START 0; "(...) ONTRACT.description" | REQUESTOR_BEGIN_OF_CONTRACT 0; "(...) UMBER.description" | REQUESTOR_CELLULAR_NUMBER 0; "(...) _PATH.description" | REQUESTOR_DEPARTMENT_PATH 0; "(...) D.description" | REQUESTOR_EMPLOYEE_ID 0; "(...) cription" | REQUESTOR_E_MAIL 0; "(...) description" | REQUESTOR_LAST_NAME 0; "(...) ATH.description" | REQUESTOR_LOCATION_PATH 0; "(...) ER.description" | REQUESTOR_PHONE_NUMBER 0; "(...) F.description" | RISK_DESCRIPTION_HREF 0; "(...) escription" | TIME_USED_TO_DELIVER_FEEDBACK 0; "(...) e_actions.description" | delete_actions 0; "(...) erties.filedata.description" | filedata 0; "(...) erties.filename.description" | filename 0; "(...) on" | action_type_id 0; "(...) tion" | contractual_cost 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.description" | folders 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.hidden.title" | folders hidden 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.iconId.title" | folders iconId 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.id.description" | id 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.id.title" | folders id 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.index.title" | folders index 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.folders.items.properties.name.title" | folders name 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.hidden.description" | hidden 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.iconId.description" | iconId 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.index.description" | index 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.name.description" | name 0; "paths./AtanorPortalAPI/atanor/project/.get.responses.200.schema.properties.scenarios.description" | scenarios 0; "paths./AtanorPortalAPI/atanor/projects/.get.responses.200.schema.properties.domainList.description" | domainList 0; "paths./AtanorPortalAPI/atanor/projects/.get.responses.200.schema.properties.domainList.items.properties.id.title" | domain id 0; "paths./AtanorPortalAPI/atanor/projects/.get.responses.200.schema.properties.domainList.items.properties.name.title" | domain name 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.availableScenarios.description" | availableScenarios 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.bestRank.description" | bestRank 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.domainId.description" | domainId 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.locale.description" | locale 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.pattern.description" | pattern 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.scenarios.items.properties.iconId.title" | scenarios iconId 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.scenarios.items.properties.id.title" | scenarios id 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.scenarios.items.properties.name.title" | scenarios name 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.scenarios.items.properties.rank.title" | scenarios rank 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.searchTimeMs.description" | searchTimeMs 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.searchUid.description" | searchUid 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.userId.description" | userId 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.userLogin.description" | userLogin 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.versionId.description" | versionId 0; "paths./AtanorPortalAPI/atanor/search/.get.responses.200.schema.properties.worstRank.description" | worstRank 0; "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.email.description" | email 0; "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.expert.description" | expert 0; "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.firstName.description" | firstName 0; "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.grps.description" | grps 0; "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.lastName.description" | lastName 0; "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.properties.description" | properties 0; "paths./AtanorPortalAPI/atanor/user/.get.responses.200.schema.properties.sessionLess.description" | sessionLess

@ezv-compteintegration for example the first one would be line 208 and 209 in the swagger file. 
![image](https://github.com/microsoft/PowerPlatformConnectors/assets/97551789/d619af26-f0a0-49f8-901b-952cffd324a9)

Amjed-Ayoub commented on certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,17505 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Different endpoints for creating/updating/deleting in the Easyvista Service Manager database / EasyVista Service Manager Modification May 2023",
```
@ezv-compteintegration is it possible to remove "/ EasyVista Service Manager Modification May 2023" from the description?

ezv-compteintegration replied:
@Amjed-Ayoub  yes it's ok now

ezv-compteintegration committed:
Update apiDefinition.swagger.json

ezv-compteintegration committed:
Update apiDefinition.swagger.json

ezv-compteintegration commented:
@Amjed-Ayoub I modified all the strings with "$".
Are there any other changes?
Thanks

ezv-compteintegration commented:
@Amjed-Ayoub is it ok since my last message ?
Thanks

Amjed-Ayoub reviewed (approved):
Hello @ezv-compteintegration ,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

Amjed-Ayoub reviewed (approved):
Hello @ezv-compteintegration ,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

ezv-compteintegration commented:
@Amjed-Ayoub i have submit the files to ISV Studio yesterday

Amjed-Ayoub commented:
> @Amjed-Ayoub i have submit the files to ISV Studio yesterday

Thank you so much @ezv-compteintegration. Our engineers will review very soon.

cedric-martin commented:
Hello,
i find 3 errors after testing the new connector with the team of ISV Studio.
So i change apiDefinition.swagger.json, now what is the procedure to change_it and validate this new file ?
thanks
